font-codegen/src/parsing.rs

```diff
@@ -519,6 +534,24 @@ impl Parse for RawEnum {
     }
 }
 
+impl Parse for ExternTarget {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
+        let _kw = input.parse::<Token![extern]>()?;
+        let target = match input.parse::<syn::Ident>()?.to_string().as_str() {
+            "scalar" => ExternTarget::Scalar(input.parse()?),
+            "record" => ExternTarget::Record(input.parse()?),
+            _ => {
+                return Err(syn::Error::new(
+                    input.span(),
+                    "expected one of 'scalar' 'record'.",
+                ))
+            }
+        };
```

Comment:
instead of this, we can use [custom keywords](https://docs.rs/syn/latest/syn/macro.custom_keyword.html). We already have a custom keyword for 'record', and we could add one for 'scalar'; search for "mod kw" in this file.

Reply:
Done, ty